src/es/escaneodeleyendas/build.gradle

```
@@ -0,0 +1,9 @@
+ext {
+    extName = 'Escaneo de Leyendas'
+    extClass = '.EscaneoDeLeyendas'
+    themePkg = 'madara'
+    baseUrl = 'https://escaneodeleyendas.com'
+    overrideVersionCode = 0
```

Set NSFW explicitly.
```suggestion
    overrideVersionCode = 0
    isNsfw = true
```
```suggestion
    overrideVersionCode = 0
    isNsfw = false
```